tests/meltano/cli/test_install.py

```diff
@@ -242,3 +244,12 @@ def test_clean_install(
 
             mappers = [m for m in commands[0][1] if m == mapper]
             assert len(mappers) == 3
+
+
+# TODO is install_service the right place for this?
+def test_new_folder_should_autocreate_on_install(project, cli_runner, un_engine_uri):
+    # We had a case defined https://github.com/meltano/meltano/issues/6383 that caused the meltano db to not be recreated
+    shutil.rmtree(".meltano")
+    result = cli_runner.invoke(cli, ["install", "--clean"])
```

Comment:
The `.meltano` directory is being recreated in the temp directory, but there's no database being created.

I tested with the config command below `meltano config meltano list` and the URI is set properly (SQL Lite DB in the Temp Directory) not sure what's going on here